app/models/location.rb

```diff
@@ -35,7 +35,7 @@ class Location < ActiveRecord::Base
   validate :detailed_hours_cannot_end_before_start
 
   attr_accessible :region_id, :address, :twitter_handle, :admin_notes, :contact, :donor_type, :hours,
-                  :is_donor, :lat, :lng, :name, :public_notes, :recip_category, :website, :receipt_key,
```

Comment:
There's an [`is_donor` method defined on line 42](https://github.com/ericfreese/food-rescue-robot/blob/4e2c3ab0553c82b614df575f2fb05c700879a940/app/models/location.rb#L42-L44), but the database column was removed with commit 13a22f6.